app/helpers/batch_edits_helper.rb

```diff
@@ -6,6 +6,6 @@ def batch_delete
   end
 
   def render_check_all
-    render partial: 'batch_edits/check_all' unless params[:controller].match("my/collections")
+    render partial: 'batch_edits/check_all' unless params[:controller] =~ /my\/collections/
```

Comment:
Well, it looks like it works OK in this case. I wonder what caused the bug in Sufia 7. :ghost: